api/src/shared/application/scripts/parsers.js

```diff
@@ -27,6 +29,68 @@ export function csvFileParser(columnSchema = []) {
   };
 }
 
+/**
+ * Stream CSV file by chunk with the given column schema
+ * @param {Record<string, Joi.Schema>} columnSchema
+ * @returns {Promise<Function>} function to process each chunk
+ */
+export function csvFileStreamer(columnSchema = []) {
```

Comment:
C’est du pinaillage sur le nommage, mais ça serait vraiment plus clair si cette variable s’appelait `columnSchemas`, ce qui voudrait dire _« les schémas de colonnes »_ : 
```suggestion
export function csvFileStreamer(columnSchemas = []) {
```

Dans le code actuellement on trouve un grand nombre de `columnsSchema` particulièrement mal nommés, c'est de l'anglais incorrect. Mais ça vient sûrement du fait que les devs voulaient un `s` quelque part pour bien avoir en tête que c'est un tableau d’éléments.

Et si vous êtes d'accord avec cette proposition de nommage on pourrait ajouter un petit commit de normalisation à la fin de cette PR qui ferait la normalisation partout dans le code de `api/` : 
* `columnsSchemas`→`columnSchemas`
* `columnsSchema`→`columnSchemas`
* `columnSchema`→`columnSchemas`